src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -525,8 +525,8 @@ def _evaluation_detector_conditions(
                 if condition_result.result is not None
                 and isinstance(condition_result.result, DetectorPriorityLevel)
             ]
-
-            new_priority = max(new_priority, *validated_condition_results)
+            if validated_condition_results:
+                new_priority = max(new_priority, *validated_condition_results)
```

Comment:
This was throwing an error when `validated_condition_results` is an empty list because either an error was raised in `create_occurrence` or an occurrence was not created because the threshold was not met.

Reply:
thanks for the fix 🙏